.github/policies/resourceManagement.yml

```diff
@@ -205,7 +205,47 @@ configuration:
       - addReply:
           reply: This issue has been marked as "Waiting on Author" and has not had any activity for **7 day**. It has been closed for housekeeping purposes.
       - closeIssue
+    - description: Resolution no activity 
+      frequencies:
+      - hourly:
+          hour: 12
+      filters:
+      - isOpen
+      - isIssue
+      - hasLabel:
+          label: Resolution-No Activity
+      - noActivitySince:
+          days: 7
+      actions:
+      - addReply:
+          reply: This issue has been marked as "No Activity". It has been closed for housekeeping purposes.
+      - closeIssue
+    - description: This is for closing issues that have no activity for 6 months.
+      frequencies:
+      - hourly:
+          hour: 24
+      filters:

```

Comment:
Can we add a tag to keep issues open if the team decides we want them open for longer?

Reply:
Yes we can create an label something like "Keep Open" and then add that to the Resolution no activity filter.

Reply:
Created label `KeepOpen`